test_libs/gen_helpers/gen_base/gen_suite.py

```diff
@@ -0,0 +1,17 @@
+from typing import Iterable
+
+from eth_utils import (
+    to_dict,
+)
+
+from gen_base.gen_typing import TestCase
+
+
+@to_dict
+def render_suite(*, title: str, summary: str, fork: str, config: str, test_cases: Iterable[TestCase]):
+    yield "title", title
+    if summary is not None:
+        yield "summary", summary
+    yield "fork", fork
+    yield "config", config
+    yield "test_cases", test_cases
```

Comment:
Oops, just noticed, if we allow test cases to be iterables this should probably become something like
```suggestion
    yield "test_cases", tuple(test_cases)
```